yellowbrick/features/pcoords.py

```diff
@@ -210,6 +212,17 @@ class ParallelCoordinates(DataVisualizer):
     n_samples_ : int
         number of samples included in the visualization object
 
+    features_ : ndarray, shape (n_features,)
+        The names of the features discovered or used in the visualizer that
+        can be used as an index to access or modify data in X. If a user passes
+        feature names in, those features are used. Otherwise the columns of a
+        DataFrame are used or just simply the indices of the data array.
+
+    classes_ : ndarray, shape (n_classes,)
+        The class labels that define the discrete values in the target. Only
+        available if the target type is discrete. This is guaranteed to be
+        strings even if the classes are a different type.
+
```

Comment:
I'm a bit worried about consistency in the docstring of ParallelCoordinates, RadViz, and DataVisualizer ... then eventually in PCA and Manifold as well. Any advice would be welcome!

Reply:
Good question... perhaps we should open up a follow-on task. @Kautumn06 &mdash; thoughts? You're always so good at helping with consistency in our communications across similar visualizers!